spiped/main.c

```diff
@@ -274,50 +281,58 @@ main(int argc, char * argv[])
 			warnp_syslog(1);
 	}
 
+	/* Resolve bind address. */
+	if (opt_b) {
+	    if ((sa_b = sock_resolve(opt_b)[0]) == NULL) {
+		warn0("No address found for %s", opt_b);
+		goto err2;
+	    }
+	}
```

Comment:
Indentation mixes tabs and spaces here.

Reply:
Right.

I also wonder whether it should be
```
if (opt_b && ((sa_b = sock_resolve(opt_b)[0]) == NULL)) {
```

My preference would be to separate the `(opt_b)` from the `sock_resolve()`, but I suspect that Colin might prefer it all on one line.

(@nerijus: I wouldn't bother changing this part until Colni comments)

Reply:
Rewrote it.